invoke-ai/InvokeAI
# fix(ui): reset canvas batchIds on clear/batch cancel

## What type of PR is this? (check all applicable)


## Description

This was in the original fix in #4829 but I must have removed it accidentally.

## Related Tickets & Documents



- Related Issue #
- Closes #4889

## QA Instructions, Screenshots, Recordings

- Start from a fresh canvas session (may need to let a generation finish or reset web UI if yours is locked)
- Invoke/add to queue
- Immediately cancel current, clear queue, or clear batch (can do this from the queue tab)
- Canvas should return to normal state
